static/app/views/starfish/views/appStartup/index.tsx

```diff
@@ -56,7 +70,11 @@ function AppStartup({additionalFilters}: Props) {
     query.addStringFilter(prepareQueryForLandingPage(searchQuery, false));
   }
 
-  const queryString = appendReleaseFilters(query, primaryRelease, secondaryRelease);
+  const queryString = `${appendReleaseFilters(
+    query,
+    primaryRelease,
+    secondaryRelease
+  )} (count_starts(measurements.app_start_cold):>0 OR count_starts(measurements.app_start_warm):>0)`;
```

Comment:
Is the intention of this to filter out screens that don't have cold warm starts? I think we will run into problems with HAVING clauses when we implement the metrics layer. Not great, but okay for now. We will probably want switch over to the spans dataset by then and can just filter to the right span op and group by segment names - which is better because it's just a WHERE filter on those tags.

Also, this condition only gets applied after the merges are resolved, so the `Total Count` here isn't the total of just cold start and warm start screens - it's the actual total count of the screen (ie, the intermediate aggregation buckets with 0 cold start and warm start counts don't get dropped). Not sure if it was also the intention of this filter to get "accurate" total counts. 
![Screenshot 2023-12-12 at 11 38 39 AM](https://github.com/getsentry/sentry/assets/63818634/95374a6c-ce74-466c-a055-3ec85896dbea)

Reply:
Oof, okay this is really good feedback. That totally was my intention. I didn't know this would be applied after the merges are resolved and therefore this leads to inaccurate counts. Since `app.start.cold` and `app.start.warm` are being indexed already, there's nothing preventing me from converting this today is there? If not, I'll merge this as-is for now but follow up with a PR to convert datasets.

This was a great comment and had me think deeper about how I'm querying for this data, really appreciate it!